src/components/TileVideo/index.js

```diff
@@ -25,7 +25,7 @@ export default class TileVideo extends PureComponent {
     autoPlay: false,
     controls: false,
     loop: false,
-    muted: false,
+    muted: true,
```

Comment:
Intended change? All videos now muted by default?

Reply:
ha, nice catch.  i must have set that while i was coding so that i could continue to listen to music 😂

Reply:
jk, this is intentional.  thought this was on the `VideoPlayer`, but yes... `VideoTile` should be muted by default